### PR TITLE
♻️Maintenance: mypy simcore sdk

### PR DESCRIPTION
**Pull request by sanderegg**
## What do these changes do?

- relates to https://github.com/ITISFoundation/osparc-simcore/issues/6115

## Dev-ops checklist

- [x] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

File: packages/simcore-sdk/src/simcore_sdk/node_ports_v2/links.py [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
-from models_library.basic_regex import UUID_RE
 from models_library.projects_nodes_io import BaseFileLink, DownloadLink
 from models_library.projects_nodes_io import PortLink as BasePortLink
-from pydantic import AnyUrl, Extra, Field, StrictBool, StrictFloat, StrictInt, StrictStr
+from pydantic import AnyUrl, Extra, StrictBool, StrictFloat, StrictInt, StrictStr
 
 
 class PortLink(BasePortLink):
-    node_uuid: str = Field(..., regex=UUID_RE, alias="nodeUuid")
+    ...
```
Comment (GitHK) [inline packages/simcore-sdk/src/simcore_sdk/node_ports_v2/links.py:10]: Please double check that this is correct. Looks suspicious to me

**Review by GitHK — APPROVED** [review]
👍

File: packages/simcore-sdk/src/simcore_sdk/node_ports_v2/links.py (hunk 1 above)
Comment (sanderegg) [reply packages/simcore-sdk/src/simcore_sdk/node_ports_v2/links.py:10]: yes it is suspicious. but BasePortLink already defines node_uuid.

**Review by matusdrobuliak66 — APPROVED** [review]

File: packages/simcore-sdk/src/simcore_sdk/node_ports_v2/links.py (hunk 1 above)
Comment (sanderegg) [reply packages/simcore-sdk/src/simcore_sdk/node_ports_v2/links.py]: reverted, but this code is ugly

**Pull request merged**